tox.ini

```diff
@@ -1,14 +1,10 @@
 [tox]
 envlist =
-    # Python 2.7 support was removed in Sphinx 2
-    py{27}-sphinx{17,18}{-html4,-html5,}{-qa,}
-    py{36,37,38,39}-sphinx{18,20,21,22,23,24,30,31,32,33,34,35,40,41,42,43,44,45,50,51,52}{-html4,-html5,}{-qa,}
-    # Python 3.10 working from Sphinx 4.2 and up
-    py{310}-sphinx{42,43,44,45,50,51,52,53,latest}{-html4,-html5}{-qa,}
-    # Sphinx 6+ has simplified docutils and Python support
-    py{38,39,10}-sphinx{60,61,70,71}{-html5,}{-qa,}
+    py{38,39,310}-sphinx{50,51,52,53,60,61}{-html4,-html5,}
+    # HTML4 is not support on Sphinx >=7.x
+    py{38,39,310}-sphinx{70,71,latest}{-html5,}
```

Comment:
There is a conversation about what we really need to be testing here too. While it's helpful to run these through CI, I think visually testing 3.6 - 3.10 is not helpful. I'd love to have a better policy here and think we could say something like we visually test against the latest Python only? This reduces our support matrix a great deal.

Same feedback here though, we should be running all of our supported Pythons through CI at least.